### ISSUE TITLE
The "Join Video/Meetecho" button on the agenda page should be larger/brighter than the other buttons

### ISSUE THREAD

**Issue by mirjak** [ietf-tools/datatracker#4288]
### Description

This would make it much easier to find the button that probably many people use most during the meeting.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by larseggert** [comment]
Would splitting those out into a separate button group work? See below:
<img width="285" alt="Screen Shot 2022-07-28 at 10 52 51" src="https://user-images.githubusercontent.com/200328/181568340-47455358-1d93-4681-a537-ede2b96bd339.png">

**Comment by mirjak** [comment]
Yes, much better. I guess we could still highlight them more, e.g. different colour?
